src/vlinder/make_report.py

```diff
@@ -0,0 +1,234 @@
+        """
+        if "title_" + target in self.input_dict["case_text_elements"]:
+            text_element = "case_text_element"
+        else:
+            text_element = "generic_text_element"
+        if "title_" + target in self.input_dict[text_element + "s"]:
+            if str(
+                self.input_dict[text_element + "_value"][
+                    list(self.input_dict[text_element + "s"]).index("title_" + target)
+                ]
+            ) != str("nan"):
+                text = (
+                    self.input_dict[text_element + "_value"][
+                        list(self.input_dict[text_element + "s"]).index("title_" + target)
+                    ]
+                    + scenario
+                    + pos_series
+                    + key_output
+                )
+            else:
+                text = "Not defined in template"
+
+        else:
+            text = "Not defined in template"
+        return text
+
+    def make_strategic_challenge(self) -> str:
+        """
+        This function adds an introduction to a slide
+        :return: the strategic challenge
+        """
+
+        if "strategic_challenge" in self.input_dict["case_text_elements"]:
+            if str(
+                self.input_dict["case_text_element_value"][
+                    list(self.input_dict["case_text_elements"]).index("strategic_challenge")
+                ]
+            ) != str("nan"):
+                text = self.input_dict["case_text_element_value"][
+                    list(self.input_dict["case_text_elements"]).index("strategic_challenge")
+                ]
+            else:
+                text = "Not defined in template"
+        else:
+            text = "Not defined in template"
+        return text
+
+    def make_introduction(self, target) -> str:
+        """
+        This function adds an introduction to a slide
+        :param target: the desired slide where the title is wanted
+        :return: the desired introduction
+        """
+
+        if "intro_" + target in self.input_dict["case_text_elements"]:
+            text_element = "case_text_element"
+        else:
+            text_element = "generic_text_element"
+        if "intro_" + target in self.input_dict[text_element + "s"]:
+            if str(
+                self.input_dict[text_element + "_value"][
+                    list(self.input_dict[text_element + "s"]).index("intro_" + target)
+                ]
+            ) != str("nan"):
+                text = self.input_dict[text_element + "_value"][
+                    list(self.input_dict[text_element + "s"]).index("intro_" + target)
+                ]
+            else:
+                text = "Not defined in template"
+
+        else:
+            text = "Not defined in template"
+        return text
+
+    def make_slides_pdf(self, scenario, orientation):
+        """
+        This function present different visualisation on different slides in a PDF format
```

Comment:
This function present**s** different visualisation**s** on different slides in a PDF format
        :param scenario: the desired scenario which is used in the visualisations
        :param orientation: the desired orientation for PDF format; there is a choice between Portrait **or** Landscape
        :return: the created **p**resentation